src/shogun/classifier/svm/LibLinear.cpp

```diff
@@ -56,6 +56,7 @@ void CLibLinear::init()
 	C2=1;
 	set_max_iterations();
 	epsilon=1e-5;
+	set_compute_bias(false);
```

Comment:
Actually Liblinear has a variable called `use_bias` to determine if the bias shall be used and it's mentioned in the comments. I can add a comment like `prevent default bias computation`